/**: # Global Params
  ros__parameters:
      traj_name: "buggycourse_nand.json"

NAND:
  bnyahaj:
    ros__parameters:
      teensy_name: "ttyUSB0"

  NAND_controller:
    ros__parameters:
      dist: 0.0
      # traj_name: "buggycourse_safe.json"
      controller: "stanley"
